plugin/forward/health.go

```diff
@@ -62,7 +62,7 @@ func (h *dnsHc) GetRecursionDesired() bool {
 }
 
 func (h *dnsHc) SetDomain(domain string) {
-	h.domain = domain
+	h.domain = dns.Fqdn(domain)
```

Comment:
This should instead be done with plugin.Name/plugin.Normalize, and error if not OK in setup.go parseBlock

Reply:
OK, so we will check this domain in setup.go, and if it is not a fqdn, then return error for it